### PR TITLE
I too (2) can now see (C) multiple bytes

### PR DESCRIPTION
**Pull request by pro-sper**
readReg() method modified for multibyte i2c reads

### CODE REVIEW

**Comment by pro-sper** [comment]
[I2C Read Multiple Bytes](https://trello.com/c/wik1gcJI/120-i2c-read-multiple-bytes)

File: src/io/I2C.cpp [hunk 1]
```diff
@@ -42,17 +42,17 @@ I2C::I2CStatus I2C::writeReg(uint8_t addr, uint8_t* reg, uint8_t regLength,
...
- * of 8 bytes starting with the MSB. Reading the response is also done
+ * of 8 bytes starting with the MSB. Reading the response is also done,
  * starting with the LSB.
  */
 I2C::I2CStatus I2C::readReg(uint8_t addr, uint8_t* reg, uint8_t regLength,
                             uint8_t* bytes, uint8_t length) {
     // Write out register address
-    for (int i = 0; i < regLength; i++)
-        I2C_RETURN_IF_ERR(write(addr, reg[i]));
+    I2C_RETURN_IF_ERR(write(addr, reg[0]));
```
Comment (mjmagee991) [inline src/io/I2C.cpp:51]: This write call is only writing 1 byte of the register address. You should use the write override that writes multiple bytes

File: src/io/I2C.cpp [hunk 2]
```diff
@@ -42,17 +42,17 @@ I2C::I2CStatus I2C::writeReg(uint8_t addr, uint8_t* reg, uint8_t regLength,
...
+
     // Read in response
-    for (int i = 0; i < length; i++)
-        I2C_RETURN_IF_ERR(read(addr, &bytes[i]));
+    I2C_RETURN_IF_ERR(read(addr, &bytes[0], length));
```
Comment (mjmagee991) [inline src/io/I2C.cpp:54]:
```suggestion
    I2C_RETURN_IF_ERR(read(addr, bytes, length));
```
Deleting these characters doesn't actually change the behavior of this call. With the ampersand and brackets, you're passing the address of the first element in the array, which is what `bytes` already is

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
A couple minor changes

**Review by mjmagee991 — APPROVED** [review]
Nice

**Review by ishan-renjen — APPROVED** [review]

**Review by ActuallyZach — APPROVED** [review]

**Pull request merged**